src/array/list/iterator.rs

```diff
@@ -13,7 +13,7 @@ pub struct ListValuesIter<'a, A: IterableListArray> {
 
 impl<'a, A: IterableListArray> ListValuesIter<'a, A> {
     #[inline]
-    pub fn new(array: &'a A) -> Self {
+    pub(crate) fn new(array: &'a A) -> Self {
```

Comment:
doesn't this need to be public for you to use it?

Reply:
We get it by calling `values_iter` on a list array. Don't think we need a `new` publicly? Don't have much opinion about it though.